query-engine/core/src/query_graph_builder/write/nested/mod.rs

```diff
@@ -32,16 +33,18 @@ pub fn connect_nested_query(
 
     for (field_name, value) in data_map {
         match field_name.as_str() {
-            "create" => nested_create(graph, parent, &parent_relation_field, value, &child_model)?,
-            "update" => nested_update(graph, &parent, &parent_relation_field, value, &child_model)?,
-            "upsert" => nested_upsert(graph, parent, &parent_relation_field, value)?,
-            "delete" => nested_delete(graph, &parent, &parent_relation_field, value, &child_model)?,
-            "connect" => nested_connect(graph, parent, &parent_relation_field, value, &child_model)?,
-            "disconnect" => nested_disconnect(graph, parent, &parent_relation_field, value, &child_model)?,
-            "set" => nested_set(graph, &parent, &parent_relation_field, value, &child_model)?,
-            "updateMany" => nested_update_many(graph, &parent, &parent_relation_field, value, &child_model)?,
-            "deleteMany" => nested_delete_many(graph, &parent, &parent_relation_field, value, &child_model)?,
-            "connectOrCreate" => nested_connect_or_create(graph, parent, &parent_relation_field, value, &child_model)?,
+            args::CREATE => nested_create(graph, parent, &parent_relation_field, value, &child_model)?,
```

Comment:
Above you had `fields::AVG` etc., and the nested operations are not really `args`. I suggest the `fields` or `operations` namespace.